python/ray/serve/autoscaling_policy.py

```diff
@@ -68,7 +68,10 @@ def __init__(self, config: AutoscalingConfig):
 
     @abstractmethod
     def get_decision_num_replicas(
-        self, current_num_ongoing_requests: List[float], curr_target_num_replicas: int
+        self,
+        current_num_ongoing_requests: List[float],
+        curr_target_num_replicas: int,
+        current_handle_queued_queries: float,
```

Comment:
can we message the order a bit? the current ordering is (metrics, config, metrics)